esmvaltool/reformat_scripts/reformat_obs_DUMMY.ncl

```diff
@@ -0,0 +1,9 @@
+;; this is a dummy test script
```

Comment:
I would put the reformat scripts in `esmvaltool/cmor/cmorize_obs/` and name them as `cmorize_obs_[dataset].ncl`.